requirements/base.txt

```diff
@@ -9,8 +9,9 @@ blinker==1.4              # via flask-sso
 click==6.7                # via flask
```

Comment:
It's mostly future-proofing, when we start adding extra requirements for testing etc. they can go in a separate requirements file from that used by the server. I'm also using pip-tools to generate pinned requirements files from input specifications, so I don't need to track dependencies manually. So the .in files are the master, and the .txt files get auto-generated when you run make.